css/settings/index.css

```diff
@@ -0,0 +1,33 @@
+@import 'colors.css';
+
+[data-theme="main"] {
+    --primary: var(--light-grey);
+    --secondary: var(--black);
+    --light: var(--white);
+    --primary-text: var(--black);
+    --secondary-text: var(--white);
+}
+
+[data-theme="danger"] {
+    --primary: var(--light-red);
+    --secondary: var(--dark-red);
+    --light: var(--white);
+    --primary-text: var(--dark-grey);
+    --secondary-text: var(--white);
+}
+
+[data-theme="info"] {
```

Comment:
@lopenchi great job! just one thing here, when we talk about a theme schemas these two: `info` and `danger` are not a "theme schema", what I mean is: there could be an `info` or `danger` color combination for a dark theme. Check [this example from mantine](https://mantine.dev/core/alert/) and try this:
- pick a color for the component
- click the theme button of the site
- click again the same button
You should notice that even though you picked a color, the same color is changed depending on the theme of the site.

Reply:
What about themes: main, light and dark?

Reply:
if I were to add a button to change the theme, I would have these options: system (here the OS defines the color scheme), light and dark. so, you the css will end up with just two color schemes: light and dark.
check this link https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme